core/src/main/scala/chisel3/experimental/dataview/package.scala

```diff
@@ -273,7 +273,7 @@ package object dataview {
   }
 
   // Safe for all Data
-  private[chisel3] def isView(d: Data): Boolean = d._parent.contains(ViewParent)
+  private[chisel3] def isView(d: Data): Boolean = d._parent.exists(_ == ViewParent)
```

Comment:
This looks equivalent but because `contains` takes its argument by value, even if `_parent` is `None`, `ViewParent` must be initialized (it's a lazy val) which requires a Chisel context (because it's a Module)... yay side effects 🤮.